grid/recruitment/create-candidate/maps/breezy-hr.test.ts

```diff
@@ -1,6 +1,53 @@
+import nock from 'nock';
+
 import { createCandidateTest } from './create-candidate';
+import { createCandidateFeaturesTest } from './create-candidate-features';
+
+createCandidateTest(
+  'breezy-hr',
+  {
+    valid: '701c3f2fc42a01',
+    invalid: '12345ab789d1dd',
+  },
+  {
+    beforeRecordingLoad: recordings => {
+      recordings.forEach(recording => {
+        if (
+          recording.method === 'POST' &&
+          /^\/v3\/company\/.+\/position\/.+\/candidate\/.+\/resume$/m.test(
+            recording.path
+          )
+        ) {
+          const boundaryReplacement = '---boundary replaced';
+          //resume upload contains multipart/form-data body with boundary with randomized value, we replace the boundary with constant in recorded body and matched body
+          const recordedBody = replaceBoundaryInMultipartFormDataBody(
+            boundaryReplacement,
+            recording.body
+          );
+          recording.body = body => {
+            const requestBody = replaceBoundaryInMultipartFormDataBody(
+              boundaryReplacement,
+              body
+            );
```

Comment:
question: why is this needed? 🤔

Reply:
The boundary needs to be replaced with constant value in recorded and tested requests because boundary changes with each subsequent request even when the rest of the request contains the same data.

This will be the case of every multipart/form-data request tested using nock and @superfaceai/testing lib. We should come with generic solution in @superfaceai/testing library.

Reply:
See my suggestion with better description in comment above.

Reply:
I understand and agree that this should be added in processing of recordings logic in testing library. I have two more questions:
- why do you assign function to a body?
- isn't there a need for `beforeRecordingSave` implementation? Does it work only with this (record -> test with recorded traffic)?

Reply:
> why do you assign function to a body?

The function is custom body matcher implementation. It is supported by nock library.

> isn't there a need for beforeRecordingSave implementation

Well we could change the body before we save the recording, but that body would still not match the body generated during the test run. Boundary is random value.

Reply:
Thanks for clarifying, was just wondering if we don't want to replace boundary in `beforeRecordingSave` and keep recording without this dynamic boundary, my only concern is that recording body will change each time we rerecord it and it is not clear what changed in hex format. 🤷🏻‍♂️

Reply:
> just wondering if we don't want to replace boundary in beforeRecordingSave and keep recording without this dynamic boundary

It is the request body that is changing, not response body. Keep recording without dynamic boundary would not help, because the request body boundary generated during test run would not match saved recording.

Reply:
Wouldn't it match recordings with your implementation of `beforeRecordingLoad` (with use of function as a body matcher)?